src/DisplayBoard.cpp

```diff
@@ -0,0 +1,110 @@
+#include "DisplayBoard.hpp"
+
+namespace
+{
+    using u_int = unsigned int;
+
+    const char HORIZONTAL = '-';
+    const char VERTICAL = '|';
+    const char INTERSECTION = '+';
+    const char PADDING = ' ';
+
+
+    std::string drawPadding(const u_int width, const char sign = PADDING);
+    std::string drawEndLine(const u_int width);
+    std::string drawEmptyLine(const u_int width, const char sign = PADDING);
+    std::string drawColumns(const u_int maxElementsInRows, const u_int colNumber, const u_int heightCol);
+    std::string drawRow(const u_int maxElementsInRows);
+    std::stringstream drawBoard(const u_int width,
+                                const u_int height,
+                                const u_int widthRows,
+                                const u_int heightCol);
+}
+
+namespace DisplayBoard
+{
+    std::string displayInterface(const Board& b){
+        auto width = b.getRowsNumber();
+        auto height = b.getColsNumber();
+        auto widthRows = b.findLongestVectorInRows();
+        auto heightCol = b.findLongestVectorInCols();
+
+        auto tmp = drawBoard(width, height, widthRows, heightCol);
+        auto tmpStr = tmp.str();
+        std::cout << tmpStr << "\n";
+
+        return tmpStr;
+    }
+}
+
+namespace
+{
+    std::stringstream drawBoard(const u_int width,
+                                const u_int height,
+                                const u_int widthRows,
+                                const u_int heightCol)
+    {
+        std::stringstream ss;
+        std::string s = drawEndLine(widthRows);
+        s.pop_back();
```

Comment:
Why do we pop_back here?

Reply:
I am not author of this function, so @majkel84 could you tell something?

Reply:
It is because drawEndLine create something like this: +---+ and we don't want to draw ending '+' sign because in next step we will draw it once again. 
If we don't do this (pop_back) we will have something like this: +----++-----+.

Reply:
Let's leave it as it is right now, because this task is blocking every other task in project and this is non-critictal area. We will open new issue to refactor drawing after merge.

Reply:
Ok, you are right...but I just have changed it in this commit: #c870f166d32893553711b8c3a0a4e30381e6ff5e